robovat/envs/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .grasp.grasp_4dof_env import Grasp4DofEnv
 from .push.push_env import PushEnv
+from .shuffle.shuffle_env import ShuffleEnv
+from .grocery.grocery_env import GroceryEnv
```

Comment:
little confused with what should be where. Would these files exist on this forked robovat?